docs/cloud/index.md

```diff
@@ -13,6 +13,17 @@ You can [join the waitlist](https://us17.list-manage.com/survey?u=2334a0f23e55fd
 
 :::
 
+## Upgrade policy
+
+All customers will automatically be upgraded to the latest minor version.
+
+Maintenance support, for major versions, will be provided for at least 18 months after a GA release, and there will be at least 6 months' notice before EOL/deprecating support.
```

Comment:
I don't understand the meaning of this sentence. What does it mean to provide maintenance support for major versions for 18 months if we upgrade to the latest version right away?